src/seen.coffee

```diff
@@ -39,22 +39,18 @@ class Seen
   constructor: (@robot) ->
     @cache= {}
 
-    @robot.brain.on 'loaded', @load
-    if @robot.brain.data.users.length
```

Comment:
@patcon AFAICT that check was designed to make sure data gets loaded from the brain even in the event that the `loaded` event already fired? But I'm pretty sure it'll never happen because all of this _should_ execute in the first tick, before any replies from Redis get processed (assuming they even come back during the first tick, which is unlikely). But we'd need someone well-versed in Hubot core to really answer this question.

FWIW I've seen lots of other Hubot modules without this check, though of course it's very possible they're all just doing it wrong :P